src/main/java/com/minecolonies/colony/ColonyView.java

```diff
@@ -45,6 +44,8 @@
 
     //  Citizenry
     private Map<Integer, CitizenDataView> citizens = new HashMap<>();
+    private Map<Integer, WorkOrderView> workOrders = new HashMap<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'workOrders' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)